src/components/PlanRoom/SaveBoard.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, ButtonGroup, Box } from '@chakra-ui/react';
+import breakpointsButton from 'styled/PlanRoom/breakpointsButton';
+import breakpointsFontSize from 'styled/PlanRoom/breakpointsFontSize';
+import { NewTable, SaveTables } from './RoomTables/Item';
+
+const saveBoard = () => (
```

Comment:
Components should start with big letter.

Reply:
will be changed